src/sagemaker_tensorflow_container/training.py

```diff
@@ -192,6 +192,10 @@ def main():
     """
     hyperparameters = framework.env.read_hyperparameters()
     env = framework.training_env(hyperparameters=hyperparameters)
+
+    if '_tuning_objective_metric' in hyperparameters:
+        env.hyperparameters['model_dir'] = env.job_name
```

Comment:
I think we should probably append `env.job_name` to the original `model_dir` rather than replacing the original value entirely. (the old version appended to `checkpoint_path`)

Reply:
Done.